src/client/main.ts

```diff
@@ -199,8 +199,8 @@ export async function findIntentsByContext(context: Context): Promise<AppIntent[
  *
  * @throws `TypeError`: If `context` is not a valid Context
  */
-export function broadcast(context: Context): void {
-    tryServiceDispatch(APIFromClientTopic.BROADCAST, {context: parseContext(context)});
+export async function broadcast(context: Context): Promise<void> {
```

Comment:
This scars me a bit, but sounded like this has been fully thrashed out?

Reply:
But as far as this story is concerned, yes, it is a settled matter.